packages/unchained-client/src/evm/arbitrum/parser/__tests__/arbitrum.test.ts

```diff
@@ -19,6 +19,10 @@ import zrxTradeEthToUsdc from './mockData/zrxTradeEthToUsdc'
 import zrxTradeUsdcToEth from './mockData/zrxTradeUsdcToEth'
 import zrxTradeUsdcToWbtc from './mockData/zrxTradeUsdcToWbtc'
 
+vi.hoisted(() => {
+  vi.stubEnv('REACT_APP_FEATURE_NFT_METADATA', 'true')
+})
+
```

Comment:
👀 separate diff?

Reply:
i was spot checking individual tests, and needed this, so went ahead and added. don't feel like it is worth to break out imo